### ISSUE TITLE
Reproduced result on FakeAVCeleb

### ISSUE THREAD

**Issue by heyi0616** [arnabk001/lipinc-pytorch#1]
Hi, thanks for your great work!

I was wondering: can the performance reported in the paper (e.g., ACC and AUC on the FakeAVCeleb test set) be reproduced using the current codebase? I tried using the author's original TensorFlow code and provided pre-trained model, but the results were quite poor on my side.

Thanks again!
